framework/src/base/NonlinearSystem.C

```diff
@@ -163,6 +163,7 @@ NonlinearSystem::NonlinearSystem(FEProblem & fe_problem, const std::string & nam
     _preset_nodal_bcs(/*threaded=*/false),
     _splits(/*threaded=*/false),
     _increment_vec(NULL),
+    _sln_diff(addVector("sln_diff", false, GHOSTED)),
```

Comment:
It doesn't need to be `GHOSTED`... we won't ever be accessing off-processor entries.